### DISCUSSION TITLE
Sidebar in Hero

### DISCUSSION THREAD

**Discussion by lazarusA** [vuejs/vitepress#4298] [Q&A]
Is there a way to setup a sidebar in the hero landing page. Similar to the astro docs? 
https://docs.astro.build/en/getting-started/

![Screenshot 2024-10-18 at 10 48 37](https://github.com/user-attachments/assets/3ea461a4-5606-429d-9578-3cb7739b25d7)

**Comment by DraTeots** [comment]
Any updates or thoughts? Maybe @lazarusA solved it somehow?
